### PR TITLE
Disable Emoji markdown plugin

### PR DESCRIPTION
**Pull request by th0r**
Fixes https://github.com/workato/docs/issues/986

image

### CODE REVIEW

File: docs/.vuepress/config.js [hunk 1]
```diff
@@ -44,6 +44,10 @@ module.exports = {
   markdown: {
     // Searches for URLs in plain text and converts them to links
     linkify: true,
+    chainMarkdown: config => {
+      // Fixes https://github.com/workato/docs/issues/986
+      config.plugins.delete('emoji');
```
Comment (kandadaboggu) [inline docs/.vuepress/config.js]: Does this mean we can't use emoji?

Comment (th0r) [reply docs/.vuepress/config.js]: Well, yes. Do we want to?

Comment (th0r) [reply docs/.vuepress/config.js]: If we want then I think there should be a way to escape them.

Comment (th0r) [reply docs/.vuepress/config.js]: @kandadaboggu seems like they don't support escaping: https://github.com/markdown-it/markdown-it-emoji/issues/11

So we'll need to wrap all false-positives with backticks (code).

Comment (kandadaboggu) [reply docs/.vuepress/config.js]: We should escape them.

Comment (th0r) [reply docs/.vuepress/config.js]: I've found a way to disable emoji aliases like `:)` but still enable full-sized emojis like `:smile:`

Comment (kandadaboggu) [reply docs/.vuepress/config.js]: Okay

**Pull request merged**